app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,87 @@
+class OrdersController < ApplicationController
+  before_action :require_login, only: [:show_seller_orders]
+
+  def show
+    @orders = Order.find(current_order.id).orderitems
+  end
```

Comment:
Where is this called? I see that you have a route for it, but I can't find a view. I also get an error when I try to render "/orders/703/". I'm trying to figure out why you'd want to set @orders to be equal to an order's orderitems.